script/validate/dco

```diff
@@ -4,7 +4,7 @@ set -eu -o pipefail
 
 if ! command -v git-validation; then
     >&2 echo "ERROR: git-validation not found. Install with:"
-    >&2 echo "    go get -u github.com/vbatts/git-validation"
+    >&2 echo "    go install github.com/vbatts/git-validation"
```

Comment:
Should have either `@latest` added, or (as it's only an information message) we can decide to revert it to keep the diff small for this branch.

Reply:
sure, it's ok